src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -109,6 +109,9 @@ def __init__(
             new_config["steps_offset"] = 1
             scheduler._internal_dict = FrozenDict(new_config)
 
+            self.text_encoder_name = text_encoder.config["architectures"].lower()
+            self.unet_name = unet.config["_class_name"].lower()
```

Comment:
same here, we don't really need it no? Also I think it doesn't correctly overwrite here?